lib/reply.js

```diff
@@ -47,20 +48,20 @@ Reply.prototype.send = function (payload) {
     return
   }
 
-  const contentType = this.res.getHeader('Content-Type')
+  const contentType = this._headers['content-type']
```

Comment:
The `Content-Type` could have been set on the `res` object by a hook or middleware. I am pretty worried about the potential complications of having response headers stored in 2 different places.

Reply:
This can be easily solved with a `reply.getHeader` that fallbacks to the basic response headers.
I tested this in my local copy and it works perfectly.
If @mcollina is ok I can send the change directly in this branch.